src/functions-and-arrays.js

```diff
@@ -1,38 +1,156 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(a,b) 
+{
+  if (a>b) {
+    return a;
+    
+  } else {
+    return b;
+    
+  }
+}
+maxOfTwoNumbers(a,b);
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(wordsArr)
+ {
+  let longestWord = "";
+  if(wordsArr.length===0)
+  { 
+    return null;
+  } 
+  else if (wordsArr.length===1)
```

Comment:
You don't need to add a specific logic in case the length of the array is 1, because the logic in the body of the else-statement is covering that.